client/pkg/fileutil/fileutil_test.go

```diff
@@ -192,3 +192,17 @@ func TestDirPermission(t *testing.T) {
 		t.Errorf("expected error, got nil")
 	}
 }
+
+func TestRemoveTempFile(t *testing.T) {
+	tmpdir, err := ioutil.TempDir(os.TempDir(), "foo")
+	f, err := ioutil.TempFile(tmpdir, "tmp")
```

Comment:
Please test the the function is not deleting too much, e.g.: 

f, err := ioutil.TempFile(tmpdir, "foobar.tmp")

is preserved.

Reply:
Done